app/src/main/java/tech/zafrani/companionforpubg/adapters/NewsAdapter.java

```diff
@@ -0,0 +1,90 @@
+package tech.zafrani.companionforpubg.adapters;
+
+
+import android.content.Intent;
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import tech.zafrani.companionforpubg.R;
+import tech.zafrani.companionforpubg.models.News;
+import tech.zafrani.companionforpubg.models.NewsItem;
+
+public class NewsAdapter extends RecyclerView.Adapter<NewsAdapter.NewsItemViewHolder>{
+    private News news = new News();
+
+    public void addNewsItem(@NonNull final NewsItem newsItem) {
+        news.add(newsItem);
+    }
+
+    @Override
+    public NewsItemViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        final View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_item_news, parent, false);
+        return new NewsItemViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(NewsItemViewHolder holder, int position) {
+        final NewsItem newsItem = news.get(position);
+        holder.bind(newsItem);
+    }
+
+    @Override
+    public int getItemCount() {
+        return news == null ? 0 : news.size();
+    }
+
+
+    public class NewsItemViewHolder extends RecyclerView.ViewHolder {
+        @BindView(R.id.row_item_image)
+        ImageView imgImageView;
+
+        @BindView(R.id.row_item_title)
+        TextView titleTextView;
+
+        @BindView(R.id.row_item_type)
+        TextView typeTextView;
+
+        @BindView(R.id.row_item_date)
+        TextView dateTextView;
+
+        @BindView(R.id.row_item_description)
```

Comment:
Don't use butterknife for view holders. You can make the fields private final and find by id in the constructor